gui/src/components/Layout.tsx

```diff
@@ -118,10 +118,6 @@ const HIDE_FOOTER_ON_PAGES = [
   "/onboarding",
   "/localOnboarding",
   "/apiKeyOnboarding",
-  "/aiderMode",
-  "/inventory",
-  "/inventory/aiderMode",
-  "/inventory/perplexityMode"
 ];
```

Comment:
dont remove these, we want to hide footers on those routes

Reply:
I didn't remove those files, I'm not sure how they were removed.